app/controllers/application_controller.rb

```diff
@@ -4,14 +4,20 @@ class ApplicationController < ActionController::Base
   self.responder = ApplicationResponder
   respond_to :html
 
+  private
+
+  protect_from_forgery with: :exception
```

Comment:
This line should not be private.
This might also eliminate one of your Rubocop warnings...